README.md

```diff
@@ -61,7 +61,14 @@ INFO handle_timeout{id=<id>}: snownet::node: Connection failed (ICE timeout)
 
 - FireZone uses relay to communicate with the portal, because of this it needs at least 2 relays 
 
+## BREAKING CHANGES
+
+### 10.17.2024 Two relay setup
+
+- Previously only one relay was installed, you need at least two relays to work properly2
```

Comment:
Perhaps link to the upstream issue here? It may also be good to define what "properly" means. As long as nothing or everything gets rebooted, things will work properly without a single one too :)

Reply:
Changed